scripts/us_bea/states_gdp/states_gdp_industry_statvars.mcf

```diff
@@ -0,0 +1,168 @@
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_11
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/11
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_21
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/21
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_22
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/22
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_23
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/23
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_31_33
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/31_33
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_JOLTS_320000
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/JOLTS_320000
```

Comment:
Does BEA really refer to the same codes as JOLTS?

Reply:
Oh interesting. JOLTS codes seems to be defined within an NAICS namespace. Do our JOLTS codes refer to these combined industries?
https://www.bls.gov/jlt/jltnaics.htm

Perhaps the question is also how we should handle these combined industries. NAICS/31_33 in the node above is currently undefined right now and will be a new node. @tjann

Reply:
Will move away from JOLTS codes to explicit aggregations. For example NAICS/321&NAICS/327-339 as opposed to JOLTS_320000. Given that these aggregations aren't too large for this particular dataset, this shouldn't be an issue.